src/lib.rs

```diff
@@ -16,30 +18,30 @@ fn panic_item_not_exist(item_id: ItemId) -> ! {
     panic!("Item with the {item_id} ID doesn't exist")
 }
 
+fn panic_zero_address_among_(who: &str) -> ! {
```

Comment:
Why the underscore at the end of the function name?
```suggestion
fn panic_zero_address_among(who: &str) -> ! {
```